libvast/src/systemd.cpp

```diff
@@ -15,6 +17,27 @@ namespace vast::systemd {
 
 #if VAST_LINUX
 
+bool connected_to_journal() {
+  auto journal_env = detail::locked_getenv("JOURNAL_STREAM");
+  if (!journal_env)
+    return false;
+  size_t device_number = 0;
+  char colon = 0;
+  size_t inode_number = 0;
+  auto parser = parsers::u64 >> parsers::ch<':'> >> parsers::u64;
+  if (parser(*journal_env, device_number, colon, inode_number)) {
```

Comment:
```suggestion
  size_t inode_number = 0;
  auto parser = parsers::u64 >> ':' >> parsers::u64;
  if (parser(*journal_env, device_number, inode_number)) {
```

Either just the char, or wrap it explicitly in `unused(<parser>)`.

Reply:
Background: a string literal has `unused` as default attribute. The `ch` parsers have a `char` as attribute and are typically only use for values to be extracted.